src/main/query_profiler.cpp

```diff
@@ -247,6 +252,10 @@ string QueryProfiler::ToString(ExplainFormat explain_format) const {
 		return ToJSON();
```

Comment:
The ToJSON implementation in `query_profiler.cpp` is different from the `JSONTreeRenderer` and they produce different output. If client code is integrated with the current json output, this will be a breaking change to use the `JSONTreeRendere` here. Thoughts?

Reply:
I moved the functionality into the `ToString()` method itself. Eventually, if we move the differences in json and querytree rendering to the tree_renderer derivatives, this this function should be able to handle it all.

Reply:
> The ToJSON implementation in `query_profiler.cpp` is different from the `JSONTreeRenderer` and they produce different output. If client code is integrated with the current json output, this will be a breaking change to use the `JSONTreeRendere` here. Thoughts?

Yes, that's one of the reasons I haven't touched it yet and I don't have an immediate answer for you because I was not involved in creating the `ToJSON` logic `query_profiler.cpp` so I don't yet know what the implications behind changing that are